cmd/utils.go

```diff
@@ -36,22 +36,22 @@ func progress() chan bool {
 	return quit
 }
 
-func load(appID string) (*deis.Client, string, error) {
-	c, err := settings.Load()
+func load(appID string) (*deis.Client, string, int, error) {
```

Comment:
we should probably document this because it's unknown what `load` is intended to load up at first glance, and what exactly the values returned represent.